packages/storybook-vue/src/stories/CustomRadioButton/CustomRadioButtonWithFieldsetAndLegend.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { action } from '@storybook/addon-actions';
+import type { Meta, StoryFn } from '@storybook/vue3';
+import { CustomRadioButton } from '@utrecht/component-library-vue';
+import { argTypes, data } from './argTypes';
+export default {
+  component: CustomRadioButton,
+  id: 'vue-custom-radio-button-with-field-set-and-legend',
+  title: 'Vue.js Component/Custom Radio Button/ Custom Radio Button With Fieldset And Legend',
+  argTypes: argTypes,
+} as Meta<typeof CustomRadioButton>;
+
+const Template: StoryFn<typeof CustomRadioButton & { radioButtons: typeof data }> = (args) => ({
+  components: { CustomRadioButton },
+  setup() {
+    return { args, options: args.radioButtons };
+  },
+  template: `
+  <form>
+  <FormFieldset>
+  <FormFieldsetLegend>Kies locatie</FormFieldsetLegend>
+  <FormField v-for="{id, label, name} in options" :key="id">
```

Comment:
I think `name` should not be in this loop, because `name` needs to be the same for all custom radio buttons in the same group. Destructure `value` instead:

`v-for="{id, label, value} in options"`

Reply:
make sense